Integrations/integration-DemistoRESTAPI.yml

```diff
@@ -119,6 +119,34 @@ script:
         }
     };
 
+    var deleteIncident = function(ids_to_delete) {
+        var body = {
+            ids: ids_to_delete,
+            all: false,
+            filter: {}
+        };
+
+        var args = {
```

Comment:
no need to define args and then pass args.uri and args.body, you can just pass the uri as a string and the body already you have defined a variable above

Reply:
and it is a bad practice to shadow a variable. `args` is a global variable of demisto